### ISSUE TITLE
Kodi 

### ISSUE THREAD

**Issue by babsi0995** [s0faking/plugin.video.orftvthek#101]
Hi, currently under Kodi Leia, it is not possible to open the "live" category. It throws the following error (see attachment)
Is it possible to fix this issue in the Leia Version? I currently have plugin version 10.3?

[kodi.log](https://github.com/s0faking/plugin.video.orftvthek/files/6409788/kodi.log)

**Comment by s0faking** [comment]
This is caused by the new DRM Livestream. Use 0.12.0 from the release page. https://github.com/s0faking/plugin.video.orftvthek/releases/download/0.12.0/plugin.video.orftvthek-0.12.0.zip

Might take a while until this release gets pushed to the official repo. 

You will have to enable the inputstream helper addon manually (you'll find it under "Program addons") Also you might want to switch the Addon quality setting to "adaptive"

**Comment by babsi0995** [comment]
Yes, i was already supposing something like this. I tried to do a zip installation with your actual release. But it says it "could not resolve kodi-six 0.1.3.1." If I am correct kodi-six 0.1.3.1 is for kodi matrix right? Or is at this release just the hardcoded version checking incorrect?

**Comment by s0faking** [comment]
you're right. i've updated the file.

**Comment by babsi0995** [comment]
Thanks, now the plugin is working fine under Kodi 18.9

**Comment by prettyv** [comment]
@s0faking Sorry for hijacking a closed issue, but do you happen to know whether there was an official reason given as to why DRM was introduced for livestreams recently?

**Comment by s0faking** [comment]
@prettyv i dont know any details but i guess this is part of the whole orf player initiative. the goal for is to create a VOD platform like the bbc (iplayer) has i think (they are merging flimmit into the new site for example). not sure what the current status is on that (corona didnt help ;)), but the DRM livestream implementation is tagged as "Teststream", so i think for now they are just trying things out. the DRM stream is only enabled for ORF3 and ORF Sport+ last time i checked. you can find a few more bits of information on the APA site: https://www.ots.at/t/orf-player

**Comment by prettyv** [comment]
I see, thanks! I knew about the ORF Player initiative, so I guessed as much, but seems as usual changes are just rolled out piecemeal for better or worse - still disappointed that their web player got slower and more glitchy a few months ago and remains so when it was working very smoothly before. Interesting about the "Teststream", though I saw the doorhanger asking me to enable DRM on ORF1 and ORF2 Livestreams as well, but not on ORF Sport+ currently. Oh well, somewhat annoying while I continue to be stubborn about not wanting to enable any DRM, when the same content is available just fine after it aired. We'll see what the future brings I guess.
